### PR TITLE
Moved Switch from experimental to components folder

### PR DESCRIPTION
**Pull request by lawrencewin**
### Platforms Impacted
- [ ] iOS
- [x] macOS
- [x] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes

Remade EJ's PR because merging was too time consuming. 

Moved Switch from experimental to components folder, publishing as V1

### Verification

Did manual testing of Switch component in tester on Win32, and also ran yarn test on full repo to verify all tests still pass.

### Pull request checklist

This PR has considered (when applicable):
- [x] Automated Tests
- [x] Documentation and examples
- [x] Keyboard Accessibility
- [x] Voiceover
- [x] Internationalization and Right-to-left Layouts

### CODE REVIEW

**Comment by Saadnajmi** [comment]
Can we close https://github.com/microsoft/fluentui-react-native/pull/2160 after this merges?

**Review by Saadnajmi — APPROVED** [review]

**Review by rurikoaraki — APPROVED** [review]

File: packages/libraries/core/package.json [hunk 1]
```diff
@@ -41,6 +41,7 @@
     "@fluentui-react-native/persona-coin": "0.12.29",
     "@fluentui-react-native/pressable": "0.9.65",
     "@fluentui-react-native/radio-group": "0.16.42",
+    "@fluentui-react-native/switch": "0.8.36",
```
Comment (rurikoaraki) [inline packages/libraries/core/package.json:44]: nit: we may want to leave this out since I think it will bloat people's packages downstream.

**Comment by rohanprasadofficial** [comment]
Thanks for taking this up @lawrencewin :)

**Comment by Saadnajmi** [comment]
Woo! Glad this happened :)

**Pull request merged**